priv/repo/migrations/20190501154649_transition_table_bids_timestamps.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Renaissance.Repo.Migrations.TransitionTableBidsTimestamps do
+  use Ecto.Migration
+
+  def up do
+    rename table(:bids), :inserted_at, to: :created_at
```

Comment:
I'm curious why it was necessary to change the conventional `inserted_at` and `updated_at` for bids...?

Reply:
^^

Reply:
I will remove this. 

_[I had changed it because, in my mind, having only one timestamp–`created_at`–makes it clearer that the entries in `bids` will not be modified, which contrasts with the entries in `users` & `auctions`.]_

Reply:
I think that's fine and makes sense.